hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/logic/StandardProcessLogic.java

```diff
@@ -81,12 +87,28 @@ public StandardProcessLogic(
         this.sigImpactHistorian = sigImpactHistorian;
         this.recordStreaming = recordStreaming;
         this.workingView = workingView;
+        this.recordCache = recordCache;
     }
 
     @Override
-    public void incorporateConsensusTxn(ConsensusTransaction platformTxn, long submittingMember) {
+    public void incorporateConsensusTxn(
+            @NonNull final ConsensusTransaction platformTxn,
+            final long submittingMember,
+            @NonNull final SoftwareVersion softwareVersion) {
+
         try {
             final var accessor = expandHandleSpan.accessorFor(platformTxn);
+            // Check if the transaction is from an older event version.
+            // If so, return and set the status on the receipt to BUSY
+            if (!SEMANTIC_VERSIONS.deployedSoftwareVersion().equals(softwareVersion)) {
+                log.info(
```

Comment:
Imo we don't want to log at `info` in a non-exceptional `handle()` code path that could feasibly happen thousands of times.

Reply:
Fixed. Thanks